src/blockchain/config.ts

```diff
@@ -145,6 +146,8 @@ const kovan: NetworkConfig = {
   etherscan: {
     url: 'https://kovan.etherscan.io',
   },
+  // taxProxyRegistries: ['0x383a7fc29edde64aec7f776e2517ec8819e147f1']
```

Comment:
Why do we have one commented value and one that is actually use? If you don't need it just remove it

Reply:
Sure!